src/components/LeftSidebar/ConversationsList/Conversation.vue

```diff
@@ -95,13 +106,35 @@
 			</NcActionButton>
 		</template>
 
-		<!-- confirmation required to delete conversation -->
-		<template v-if="isDialogOpen" #extra>
-			<NcDialog :open.sync="isDialogOpen"
+		<!-- confirmation required to leave / delete conversation -->
+		<template v-if="isLeaveDialogOpen || isDeleteDialogOpen" #extra>
+			<NcDialog v-if="isLeaveDialogOpen"
+				:open.sync="isLeaveDialogOpen"
+				:name="t('spreed', 'Leave conversation')">
+				<template #default>
+					<p>{{ dialogLeaveMessage }}</p>
+					<p v-if="supportsArchive && !item.isArchived">
+						{{ t('spreed', 'You can archive this conversation instead.') }}
+					</p>
+				</template>
+				<template #actions>
+					<NcButton v-if="supportsArchive && !item.isArchived" type="secondary" @click="toggleArchiveConversation">
+						{{ t('spreed', 'Archive instead') }}
+					</NcButton>
+					<NcButton type="tertiary" @click="isLeaveDialogOpen = false">
+						{{ t('spreed', 'No') }}
+					</NcButton>
+					<NcButton type="warning" @click="leaveConversation">
+						{{ t('spreed', 'Yes') }}
+					</NcButton>
```

Comment:
![image](https://github.com/user-attachments/assets/e2f60ce2-9229-4b6e-bc21-a2c1a2ae5f2c)

flip "no" and "archive instead" ?